### ISSUE TITLE
Claiming sections

### ISSUE THREAD

**Issue by jessegmeyerlab** [jessegmeyerlab/proteomics-tutorial#54]
If you want to let people know you are working on a section please post here!

**Currently I have commitments for**
MS instrument types: Yuming Jiang and Jie Sun
Liquid separations: Simion Kreimer 

**Expressed interest**
Fragmentation: Nick Riley
Enrichment: Nick Riley
Biological Interpretation: Oliver Crook and Norbert

**Comment by yarribas** [comment]
Just learning about this nice initiative. I would be very happy to help in "protein enrichment" section if it's OK with all of you.

**Comment by lichenlady94** [comment]
I'll work on the peptide quantification part for now and will add any other sections as I finish up!

**Comment by j-berg** [comment]
If no one is working on the "Statistical hypothesis testing", I can help out with that.

**Comment by lichenlady94** [comment]
Is the biochemistry basic section complete? It seems a bit lacking, no?

**Comment by jessegmeyerlab** [comment]
> Is the biochemistry basic section complete? It seems a bit lacking, no?

If you have ideas about stuff to add we welcome additions to any section!

**Comment by rileynm** [comment]
Following up here to say I am still interested in writing/helping with fragmentation and enrichment session. @yarribas, it would be great to get your help on the enrichment section. I'll reach out to you on Twitter since I see it is linked on your GitHub profile.

**Comment by jessegmeyerlab** [comment]
To update everyone with the plan, I am waiting for a section on LC and on types of MS that I think are foundations we need to have a complete article. I hope to have them incorporated over the next month. 

Since we get closer to submitting I just wanted to check if anyone else still has sections outstanding they want to contribute?

**Comment by norbertvolkmar** [comment]
This might be a bit late, but I have started working on a small sub-section of the biological interpetation section relating to orthogonal validation of MS data that goes beyond in silico network analysis etc.. Not sure if that's (still) needed?

**Comment by jessegmeyerlab** [comment]
> This might be a bit late, but I have started working on a small sub-section of the biological interpetation section relating to orthogonal validation of MS data that goes beyond in silico network analysis etc.. Not sure if that's (still) needed?

Yes that would be great thank you! Do you think you can finish it this month?

**Comment by norbertvolkmar** [comment]
yes, i think so!

**Comment by norbertvolkmar** [comment]
@jessegmeyerlab  - I will definitely have that section ready for integration by the end of the month. Since it's a rather broad topic, it will inevitably need some refinement, but that can be done in additional rounds of editing.

**Comment by jessegmeyerlab** [comment]
Awesome thank you!

**Comment by jessegmeyerlab** [comment]
> Following up here to say I am still interested in writing/helping with fragmentation and enrichment session. @yarribas, it would be great to get your help on the enrichment section. I'll reach out to you on Twitter since I see it is linked on your GitHub profile.

@rileynm are you still planning to write fragmentation and/or enrichment?
